src/controllers/time.rs

```diff
@@ -135,9 +149,32 @@ impl TimeController {
             state.world.enemies.push(new_enemy);
         }
 
-        // Move enemies in the player's direction
+        // Move enemies in the player's direction if player is alive, otherwise let them drift in
+        // the direction they're facing
         for enemy in &mut state.world.enemies {
-            enemy.update(dt * ENEMY_SPEED + state.difficulty, state.world.player.position());
+            if !state.world.player.is_dead {
+                enemy.update(dt * ENEMY_SPEED + state.difficulty, state.world.player.position());
+            } else {
+                enemy.advance(dt * ENEMY_SPEED);
+            }
+        }
+    }
+
+    // Slowly moves stars across screen, adding and removing them when necessary
+    fn update_stars(&mut self, dt: f64, state: &mut GameState) {
+        // Advance stars
+        for star in &mut state.world.stars {
+            let speed = star.speed;
+            star.update(dt * STAR_BASE_SPEED * speed);
+        }
+
+        // Remove stars outside the viewport
+        let size = &state.world.size;
+        util::fast_retain(&mut state.world.stars, |s| size.contains(s.position()));
+
+        // Add stars up to MAX_STARS
+        while state.world.stars.len() < MAX_STARS {
+            state.world.stars.push(Star::new(state.world.size));
         }
```

Comment:
Are there any reasons why you prefer removing stars and creating them again? For me, the natural thing to do would be to let them wrap around the boundaries of the screen (like the player does).

Reply:
Just means that there won't be any stars appearing in the background - I'll change them to wrapping and we'll compare the difference.